src/Compilers/Core/Portable/RuleSet/RuleSetInclude.cs

```diff
@@ -78,11 +83,9 @@ public RuleSet LoadRuleSet(RuleSet parent)
         private string GetIncludePath(RuleSet parent)
         {
             var resolvedIncludePath = ResolveIncludePath(_includePath, parent?.FilePath);
-
-            // If we still couldn't find it then throw an exception;
             if (resolvedIncludePath == null)
             {
-                throw new FileNotFoundException(string.Format(CodeAnalysisResources.FailedToResolveRuleSetName, _includePath), _includePath);
+                return null;
```

Comment:
Agree with the premise of the change. But given there is a single caller think it would be simpler to just inline this changed method. Makes it clearer this has no other impact on the code.

Reply:
Good point, I've inlined the method.